targets/TARGET_Freescale/TARGET_KLXX/TARGET_KL05Z/device/TOOLCHAIN_ARM_MICRO/MKL05Z4.sct

```diff
@@ -4,9 +4,55 @@ LR_IROM1 0x00000000 0x8000  {    ; load region size_region (32k)
    *(InRoot$$Sections)
    .ANY (+RO)
   }
-  ; 8_byte_aligned(48 vect * 4 bytes) =  8_byte_aligned(0xC0) = 0xC0
+  
   ; 0x1000 - 0xC0 = 0xF40
   RW_IRAM1 0x1FFFFCC0 0xF40 {
    .ANY (+RW +ZI)
   }
 }
```

Comment:
Should we delete everything above this line, looks like we have 2 LR_IROM1 regions